test/runtests.jl

```diff
@@ -1,43 +1,49 @@
-using Base.Test, ImageInTerminal, Images, TestImages,  ColorTypes, FixedPointNumbers
+using Test
+using ImageInTerminal, Images, TestImages,  ColorTypes, FixedPointNumbers
+using Random
 
+@testset "ReferenceTests" begin
 # check for ambiguities
 refambs = detect_ambiguities(ImageInTerminal, Base, Core)
 using ReferenceTests
+
 ambs = detect_ambiguities(ReferenceTests, ImageInTerminal, Base, Core)
 @test Set(setdiff(ambs, refambs)) == Set{Tuple{Method,Method}}()
 
+cd(@__DIR__)
+
 # load/create some example images
 lena = testimage("lena_color_256")
 camera = testimage("cameraman")
 cameras = similar(camera, size(camera)..., 2)
-copy!(view(cameras,:,:,1), camera)
-copy!(view(cameras,:,:,2), camera)
+copyto!(view(cameras,:,:,1), camera)
+copyto!(view(cameras,:,:,2), camera)
 square = Gray{N0f8}[0.1 0.2 0.3; 0.4 0.5 0.6; 0.7 0.6 0.9]
 rgb_rect = rand(RGB{N0f8}, 2, 3)
 
 @testset "io2str" begin
-    @test_throws ArgumentError eval(@macroexpand @io2str(::IO))
+    @test_throws LoadError eval(@macroexpand @io2str(::IO))
     @test_throws ArgumentError @io2str(2)
     @test_throws ArgumentError @io2str(string(2))
     @test @io2str(print(::IO, "foo")) == "foo"
     @test @io2str(println(::IO, "foo")) == "foo\n"
     @test @io2str(show(::IO, "foo")) == "\"foo\""
     A = ones(30,30)
-    @test @io2str(show(IOContext(::IO, limit=true, displaysize=(5,5)), A)) == "[1.0 1.0 … 1.0 1.0; 1.0 1.0 … 1.0 1.0; … ; 1.0 1.0 … 1.0 1.0; 1.0 1.0 … 1.0 1.0]"
+    @test @io2str(show(IOContext(::IO, :limit => true, :displaysize => (5,5)), A)) == "[1.0 1.0 … 1.0 1.0; 1.0 1.0 … 1.0 1.0; … ; 1.0 1.0 … 1.0 1.0; 1.0 1.0 … 1.0 1.0]"
 end
 
 @testset "withcolor" begin
     @test_throws ArgumentError @withcolor throw(ArgumentError("foo"))
     @test @withcolor Base.have_color == true
-    @test @withcolor @io2str(print_with_color(:green, ::IO, "test")) == "\e[32mtest\e[39m"
+    @test_broken @withcolor @io2str(printstyled(::IO, "test", color=:green)) == "\e[32mtest\e[39m"
```

Comment:
That is not good, what is going on here?

Reply:
I've struggled with this, and haven't found a way to get the escape sequences to work, so I figured I'd leave it for a later fix.

Reply:
Fixed in subsequent commit (after consulting the manual again, don't see how I missed it before).